From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ppug <ppug@users.noreply.github.com>
Date: Tue, 1 Dec 2020 05:01:14 +1100
Subject: [PATCH] Fix default paper experience merge max value


diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
index 80b0c08cef9a5326492b1faec020929fca59ff77..2ab704b3399c0646dbd6294d7f1b1ea56337f76d 100644
--- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
@@ -357,7 +357,7 @@ public class PaperWorldConfig {
 
     public int expMergeMaxValue;
     private void expMergeMaxValue() {
-        expMergeMaxValue = getInt("experience-merge-max-value", -1);
+        expMergeMaxValue = getInt("experience-merge-max-value", 0); // PugEngine - Fix default paper experience merge max value
         log("Experience Merge Max Value: " + expMergeMaxValue);
     }
 
