From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ppug <ppug@users.noreply.github.com>
Date: Fri, 27 Nov 2020 22:08:01 +1100
Subject: [PATCH] Johnny is gay


diff --git a/src/main/java/net/minecraft/server/EULA.java b/src/main/java/net/minecraft/server/EULA.java
index b37b07e6169fb2cc67d71ee9cb3d026e33be3c90..894ddc5efc114b837fc04511a26cc7492cd292eb 100644
--- a/src/main/java/net/minecraft/server/EULA.java
+++ b/src/main/java/net/minecraft/server/EULA.java
@@ -70,7 +70,7 @@ public class EULA {
                     Properties properties = new Properties();
 
                     properties.setProperty("eula", "false");
-                    properties.store(outputstream, "By changing the setting below to TRUE you are indicating your agreement to our EULA (https://account.mojang.com/documents/minecraft_eula).\nYou also agree that pug are the cutest thing on earth.");  // Paper - fix lag; // PugEngine - Brand Change
+                    properties.store(outputstream, "By changing the setting below to TRUE you are indicating your agreement to our EULA (https://account.mojang.com/documents/minecraft_eula).\nYou also agree that pug are the cutest thing on earth. \nYou also agree that johnmmit is gay");  // Paper - fix lag; // PugEngine - Brand Change //PugEngine - Johnny is gay
                 } catch (Throwable throwable1) {
                     throwable = throwable1;
                     throw throwable1;
